### ISSUE TITLE
Can't find challenge_test_v1.json   stored in checkpoint/haotang

### ISSUE THREAD

**Issue by MsAnitaAli** [EGO4D/episodic-memory#67]
@tanghaotommy 
ReadMe.Md  of step **Preprocess - extract frames from exported clips** says:

Replace the following paths for reading and outputing to the correct place in ExtractFramesWorkflowParams in tools/preprocess/extract_ego4d_clip_frames.py:

- [ ] clip_dir: path to the downloaded clip directory  **(I have done this successfully)**

- [ ] output_dir: path to the directory where to the save the extracted frame images. **(I have done this successfully)**

- [ ] annotation_path: path to the annotation file we would like to extract frames. If you want the test set only, only extract frames for the test set. If you also would like training part, then need to extract for training set as well. **(creating problem )**

code at tools/preprocess/extract_ego4d_clip_frames.py
has this line
annotation_path: str = ("/**checkpoint/haotang**/data/EgoTracks/annotations/challenge_test_v1.json" )

**Question 1:** Where should I expect **challenge_test_v1.json** ? 
I downloaded annotations using following command
ego4d --output_directory="~/ego4d_data" --datasets annotations 
but I can't see. 

**Question 2:**  Next I would need challenge_train_v1.json and challenge_val_v1.json too. From where to get these files?
